### PR TITLE
CW+HW 20221123

### PR DESCRIPTION
**Pull request by awesomeprogrammer1**
# what

HW for class
# why

Learning Python

### CODE REVIEW

File: code_school/lists/sets_presentation.ipynb [hunk 1]
```diff
@@ -240,7 +240,15 @@
     "\n",
     "animals_which_are_mammals = animals.intersection(mammals)\n",
     "\n",
-    "print(animals_which_are_mammals)"
+    "print(animals_which_are_mammals)\n",
+    "\n",
+    "# We also have intersection_update()\n",
```
Comment (sryabkov) [inline code_school/lists/sets_presentation.ipynb:245]: Is there going to be an explanation on how `intersection_update` differs from `intersection`?

Comment (awesomeprogrammer1) [reply code_school/lists/sets_presentation.ipynb:245]: Yes Sir

File: code_school/files/email-assignments/email5_assignment4_sets.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        break
+    else:
+        base_set.intersection_update(get_unique_words(given_file))
+print(base_set)
+words_in_all_files = " ".join(base_set)
+folder_with_info = Path("code_school/files/work_files")
+word_dump = folder_with_info / "email5_assignment4_output.txt"
+file_write = open(word_dump, "w")
+file_write.write(words_in_all_files)
+file_write.close()
```
Comment (sryabkov) [inline code_school/files/email-assignments/email5_assignment4_sets.py:34]: Can you please create a function for this. Here is the signature: `def write_results(result_set: set, file_name: str) -> None:`

Comment (awesomeprogrammer1) [reply code_school/files/email-assignments/email5_assignment4_sets.py:34]: Creating function, will let know when done

Comment (awesomeprogrammer1) [reply code_school/files/email-assignments/email5_assignment4_sets.py]: Done

File: code_school/my_telegram_bot/bot.py [hunk 3]
```diff
@@ -39,7 +39,7 @@ def handle_text(message):
         bot.register_next_step_handler(user_password, registration)
     elif message.text.lower() == "authenticate":
         seconds_past_authentication = time.time()
-        if int(seconds_past_authentication) > 86400:
+        if int(time.time()) - int(seconds_past_authentication) > 86400:
```
Comment (sryabkov) [inline code_school/my_telegram_bot/bot.py:42]: Using "magic numbers" in code is a bad practice. It is much better to declare a constant (a variable that doesn't change) and use it

Compare the two code fragments below and tell which one is easier to understand:

```python
if int(time.time()) - int(seconds_past_authentication) > 86400:
```

with

```python
if int(time.time()) - int(seconds_past_authentication) > SECONDS_IN_24_HOURS:
```

Note: using all uppercase for constants is popular in many programming languages

**Pull request merged**